### PR TITLE
enable read env map from KCL option("env") function

### PR DESCRIPTION
**Pull request by satyazzz123**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):
fixes: #29 
- [ ] N
- [x] Y 

#### 2. What is the scope of this PR (e.g. component or file name):
Read the environment var map through option("env") in KRM KCL Spec.

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [ ] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

### CODE REVIEW

**Comment by satyazzz123** [comment]
Hello @Peefy I have opened this PR to address issue #29 , I have made some initial changes to read env map . Please have a look and suggest if any changes are needed. Thank you!

File: pkg/edit/bootstrap.go [hunk 1]
```diff
@@ -139,6 +146,23 @@ func constructOptions(resourceList *yaml.RNode) (*options.RunOptions, error) {
...
 		fmt.Sprintf("PATH=%s", pathOptionKCLValue),
+		// environment map example (option("env"))
+		fmt.Sprintf("env=%s", envMapOptionKCLValue),
 	}
 	return opts, nil
 }
+
+// getEnvMapOptionKCLValue retrieves the environment map from the KCL 'option("env")' function.
+func getEnvMapOptionKCLValue(resourceList *yaml.RNode) (string, error) {
+	v, err := resourceList.Pipe(yaml.Lookup("option", "env"))
```
Comment (Peefy) [inline pkg/edit/bootstrap.go:157]: Here, the environment variable os should be read through the Go function.
```go
envMap := make(map[string]string)
env := os.Environ()
for _, e := range env {
	pair := strings.SplitN(e, "=", 2)
	envMap[pair[0]] = pair[1]
}
v := yaml.Parse(envMap)
```

File: pkg/edit/bootstrap.go [hunk 2]
```diff
@@ -139,6 +146,23 @@ func constructOptions(resourceList *yaml.RNode) (*options.RunOptions, error) {
...
+	if err != nil {
+		return "", errors.Wrap(err)
+	}
+
+	envMapOptionKCLValue, err := ToKCLValueString(v, "{}")
```
Comment (Peefy) [inline pkg/edit/bootstrap.go:162]: envMapOptionKCLValue, err := ToKCLValueString(v, emptyConfig)

**Comment by satyazzz123** [comment]
@Peefy I have addressed the requested changes please review

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**